.github/workflows/release-binaries.yml

```diff
@@ -126,7 +126,7 @@ jobs:
         # add extra CMake args to disable them.
         # See https://github.com/llvm/llvm-project/issues/99767
         if [ "${{ runner.os }}" = "macOS" ]; then
-          target_cmake_flags="$target_cmake_flags -DBOOTSTRAP_COMPILER_RT_ENABLE_IOS=OFF"
+          target_cmake_flags="$target_cmake_flags -DBOOTSTRAP_COMPILER_RT_ENABLE_IOS=OFF -DLLVM_USE_STATIC_ZSTD=ON"
```

Comment:
You need to add a BOOTSTRAP_ prefix to this so that it is applied to the stage2 build.  It may also make more sense to move this into the Release.cmake file.

Reply:
thanks, trying it in Release.cmake, still scoped to only macOS (although I think there's a reasonable argument to do this on Linux as well)